test/Setup.t.sol

```diff
@@ -0,0 +1,256 @@
+
+            contractAddress[DST_CHAINS[i]][bytes("HYPERLANE_RECEIVER_ADAPTER")] =
+                address(new HyperlaneReceiverAdapter(HYPERLANE_MAILBOX));
+            
+            uint256[] memory srcChain = new uint256[](1);
+            srcChain[0] = 1;
+
+            address[] memory senderAdapters = new address[](1);
+            senderAdapters[0] = contractAddress[1][bytes("HYPERLANE_SENDER_ADAPTER")];
+
+            HyperlaneReceiverAdapter(contractAddress[DST_CHAINS[i]][bytes("HYPERLANE_RECEIVER_ADAPTER")]).updateSenderAdapter(srcChain, senderAdapters);
+        }
+        
+        vm.selectFork(fork[1]);
+
+        address[] memory _receiverAdapters = new address[](3);
+        _receiverAdapters[0] = contractAddress[56][bytes("HYPERLANE_RECEIVER_ADAPTER")];
+        _receiverAdapters[1] = contractAddress[137][bytes("HYPERLANE_RECEIVER_ADAPTER")];
+        _receiverAdapters[2] = contractAddress[42161][bytes("HYPERLANE_RECEIVER_ADAPTER")];
+
+        HyperlaneSenderAdapter(contractAddress[1][bytes("HYPERLANE_SENDER_ADAPTER")]).updateReceiverAdapter(DST_CHAINS, _receiverAdapters);
+        
+        uint32[] memory _receiverDomains = new uint32[](3);
+        _receiverDomains[0] = uint32(56);
+        _receiverDomains[1] = uint32(137);
+        _receiverDomains[2] = uint32(42161);
+
+        HyperlaneSenderAdapter(contractAddress[1][bytes("HYPERLANE_SENDER_ADAPTER")]).updateDestinationDomainIds(DST_CHAINS, _receiverDomains);
+    }   
+
+    /// @dev deploys the celer adapters to all configured chains
+    function _deployCelerAdapters() internal {
+        /// @notice deploy source adapter to Ethereum
+        vm.selectFork(fork[1]);
+        contractAddress[1][bytes("CELER_SENDER_ADAPTER")] = address(new CelerSenderAdapter(ETH_CELER_MSG_BUS));
+        
+        uint256[] memory srcChain = new uint256[](1);
+        srcChain[0] = 1;
+
+        address[] memory senderAdapters = new address[](1);
+        senderAdapters[0] = contractAddress[1][bytes("CELER_SENDER_ADAPTER")];
+
+        /// @notice deploy receiver adapters to BSC, POLYGON & ARB
+        vm.selectFork(fork[56]);
+        contractAddress[56][bytes("CELER_RECEIVER_ADAPTER")] = address(new CelerReceiverAdapter(BSC_CELER_MSG_BUS));
+        CelerReceiverAdapter(contractAddress[56][bytes("CELER_RECEIVER_ADAPTER")]).updateSenderAdapter(srcChain, senderAdapters);
+
+        vm.selectFork(fork[137]);
+        contractAddress[137][bytes("CELER_RECEIVER_ADAPTER")] = address(new CelerReceiverAdapter(POLYGON_CELER_MSG_BUS));
+        CelerReceiverAdapter(contractAddress[137][bytes("CELER_RECEIVER_ADAPTER")]).updateSenderAdapter(srcChain, senderAdapters);
+
+        vm.selectFork(fork[42161]);
+        contractAddress[42161][bytes("CELER_RECEIVER_ADAPTER")] = address(new CelerReceiverAdapter(ARB_CELER_MSG_BUS));
+        CelerReceiverAdapter(contractAddress[42161][bytes("CELER_RECEIVER_ADAPTER")]).updateSenderAdapter(srcChain, senderAdapters);
```

Comment:
nit: Could this be in a `for` loop, as done above when setting the hyperplane adapters for consistency and to make it easier as new chains are added?